blockchain/network.py

```diff
@@ -438,6 +445,9 @@ def phase_1_broadcast(self, block_info, phase_type):
                 message = template.format(type(ex).__name__, ex.args)
                 logger().warning(message)
 
```

Comment:
Might have happened during the merge with master. I deleted the duplicated lines.